src/zip.c

```diff
@@ -450,6 +450,14 @@ int zip_entry_close(struct zip_t *zip) {
     goto cleanup;
   }
 
+  // UNIX
+#if MZ_PLATFORM == 3
```

Comment:
It makes sense to have a default permission mode for entries, but:
- would be good to set it up also for other platforms (if it's doable)
- `zip_entry_close` is not a right place because it will overwrite what was set by `zip_entry_fwrite`

Reply:
It wont overwrite becouse it check current external attrs value and on UNIX for files it never will be zero.

Reply:
so maybe it makes sense to set up the default value in zip_entry_open

Reply:
I agree.

Reply:
Add a test. Unsure I can't find how to check for `#if MZ_PLATFORM == 3` in test.c. My current `#ifdef __unix__` is not correct.

Reply:
Changed. Added macOS support. I tested both OS: Linux and macOS.

Reply:
Add a test. Unsure I can't find how to check for `#if MZ_PLATFORM == 3` in test.c. My current `#ifdef __unix__` is not correct.